affinidi-messaging-mediator/conf/mediator.toml

```diff
@@ -66,6 +66,15 @@ max_queued_messages = "${MAX_QUEUED_MESSAGES:100}"
 ### Default: 10080 (7 days)
 message_expiry_minutes = "${MESSAGE_EXPIRY_MINUTES:10080}"
 
+### max_message_list_limit: Max messages retrieved 
+### when listing messages from db per request
+### Default: 100
+max_messages_list = "${MAX_MESSAGES_LIST:100}"
+
+### max_messages_to_delete: Max number of messages to be deleted per request
+### Default: 100
+max_messages_to_delete = "{MAX_MESSAGES_TO_DELETE:100}"
```

Comment:
What about `max_listed_messages` and `max_deleted_messages` to align with `max_queued_messages`?